### PR TITLE
save default values additional_zip_keys for subsequent migrations

### PR DESCRIPTION
**Pull request by 0xbe7a**
Checklist
* [x] Added a ``news`` entry

see https://github.com/conda-forge/conda-smithy/pull/1764#issuecomment-1726250148

@isuruf

### CODE REVIEW

**Comment by isuruf** [comment]
Can you add a news entry?

**Comment by 0xbe7a** [comment]
Can you elaborate on https://github.com/conda-forge/conda-smithy/pull/1764#discussion_r1330655915 ?

File: news/1769-fix-additional_zip_keys [hunk 1]
```diff
@@ -0,0 +1,23 @@
+**Added:**
```
Comment (isuruf) [inline news/1769-fix-additional_zip_keys:1]: Can you change the type to `.rst` for this file?

File: news/1769-fix-additional_zip_keys [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+* <news item>
+
+**Removed:**
+
+* <news item>
+
+**Fixed:**
+
+* Fixed additional_zip_keys, so that subsequent migrations don't break
```
Comment (isuruf) [inline news/1769-fix-additional_zip_keys:19]:
```suggestion
* Fixed additional_zip_keys, so that subsequent migrations don't break.
```

**Comment by isuruf** [comment]
Thanks

**Pull request merged**